mantidproject/mantid
# Fixes left-right parity for D33

**Description of work.**

The data from left and right front panels of D33 were swapped, this is fixed now.

Also adds parameters for summing times while merging the runs and progress reporting for `SANSILLReduction`.

**To test:**
```
SANSILLReduction(Run='075051', ProcessAs='Sample', OutputWorkspace='sample')
SANSILLIntegration(InputWorkspace='sample', CalculateResolution='None', OutputWorkspace='iq')
```
Plot the `iq` in log-log, zoom in into high Q part, there should be no discontinuity, unlike in the current `release-next`. The file is attached, chop off `.zip`, it's a nexus file.

[075051.nxs.zip](https://github.com/mantidproject/mantid/files/2960631/075051.nxs.zip)



Fixes #25125. 


*This does not require release notes since D33 is not in production use yet.*


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
